# HG changeset patch
# User xunxun1982
# Date 1411493179 -28800
#      Wed Sep 24 01:26:19 2014 +0800
# Branch MOBILE370b8_2015032422_RELBRANCH
# Node ID 1ab7f6e6cdfa186f773ee8e2c7e3d5335ea86bd4
# Parent  ec643106855d0ea4769c8184a54faf6dcf875110
libffi-better

diff -r ec643106855d -r 1ab7f6e6cdfa build/autoconf/ffi.m4
--- a/build/autoconf/ffi.m4	Sun Nov 16 15:48:21 2014 +0800
+++ b/build/autoconf/ffi.m4	Wed Sep 24 01:26:19 2014 +0800
@@ -60,12 +60,16 @@
         # Need target since MSYS tools into mozilla-build may be 32bit
         ac_configure_args="$ac_configure_args \
                            CC=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh -m64$flags\" \
-                           CXX=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh -m64$flags\""
+                           CFLAGS=\"$MOZ_OPTIMIZE_FLAGS -fp:precise -Qfast_transcendentals -GL-\" \
+                           CXX=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh -m64$flags\" \
+                           CXXFLAGS=\"$MOZ_OPTIMIZE_FLAGS -fp:precise -Qfast_transcendentals -GL-\""
         ;;
       *)
         ac_configure_args="$ac_configure_args \
                            CC=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh$flags\" \
-                           CXX=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh$flags\""
+                           CFLAGS=\"$MOZ_OPTIMIZE_FLAGS -fp:precise -Qfast_transcendentals -GL-\" \
+                           CXX=\"$_topsrcdir/js/src/ctypes/libffi/msvcc.sh$flags\" \
+                           CXXFLAGS=\"$MOZ_OPTIMIZE_FLAGS -fp:precise -Qfast_transcendentals -GL-\""
         ;;
       esac
     fi
